KDIC/BlogViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  BlogViewController.swift
+//  KDIC
+//
+//  Created by Shaun Mataire on 12/27/15.
+//  Copyright © 2015 Colin Tremblay. All rights reserved.
+//  NOTE: XCode wont let me refector this class is actully for the Blog
+
+import UIKit
+import SafariServices
+
+class BlogViewController: UIViewController,  UITableViewDataSource, UITableViewDelegate {
+
+    @IBOutlet weak var blogPostTableView: UITableView!
+    
+    var blogsDict: NSArray?
```

Comment:
The scrapper return an NSDictionary, casting it to an NSarray is easy - just an 'as'. Im not sure how to get it to [Blog](), but i'll look into it.